packages/prisma/src/prisma/migrations/20220925181035_initial_migration/migration.sql

```diff
@@ -182,6 +182,8 @@ CREATE TABLE "SessionBlock" (
 CREATE TABLE "Session" (
     "id" UUID NOT NULL,
     "isAudienceInteractionActive" BOOLEAN NOT NULL DEFAULT false,
+    "isLiveQAEnabled" BOOLEAN NOT NULL DEFAULT false,
```

Comment:
Remove this, migrations are not written by hand (with Prisma CLI, will do this at the end before merge :))

Reply:
reverted✅